src/selectors/nodes.js

```diff
@@ -257,6 +263,7 @@ export const getVisibleNodes = createSelector(
           name: nodeName[id],
           label: nodeName[id],
           fullName: nodeFullName[id],
+          icon: nodeDatasetType[id] || nodeType[id],
```

Comment:
same comment as above

Reply:
@studioswong -- as per Lim's new back-end. this works for now as MVP